[IMP] inventory: Delivery Methods

jero-odoo opened the pull request:
Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=4454218&access_token=b8742bd7-52ec-4d94-b955-96b6f2c1cc3d

Updated per SME feedback. Removed section on creating a new shipping/delivery method (this information will be added to a new doc)

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/11913.png)](https://mergebot.odoo.com/odoo/documentation/pull/11913)

jero-odoo commented:
@larm-odoo this is ready for review. Thanks!

larm-odoo reviewed (approved):
Wow- I don't have any changes or suggestions! Sadly, you'll have to re-edit this after you make the new doc, to tag all the places to the new information. But this is great as is- approving!

Felicious commented:
Hi @jero-odoo ! I agree with your decision of splitting the doc, but we shouldn’t publish it as is since users won’t be able to find the shipping method setup info until the new doc is uploaded.

Here’s the plan: 
- To ensure the doc is properly split and users can find what they need, we’ll make an exception and modify multiple files in this PR. 
- Copy the deleted info into a new doc under delivery methods, then we’ll push this PR. (2 points)
- In a follow-up PR, improve the writing and update it for version 18. (2-3 points)

Felicious reviewed (changes requested):
Hi @jero-odoo ! I agree with your decision of splitting the doc, but we shouldn’t publish it as is since users won’t be able to find the shipping method setup info until the new doc is uploaded.

Here’s the plan: 
- To ensure the doc is properly split and users can find what they need, we’ll make an exception and modify multiple files in this PR. 
- Copy the deleted info into a new doc under delivery methods (no need to improve or edit this yet), then we’ll push this PR. (2 points)
- In a follow-up PR, improve the writing and update it for version 18. (2-3 points)

jero-odoo commented:
Hey @Felicious, I added the new doc with the content and images added back in. I also replaced the links and references that were removed. Let me know if you have any questions. Thanks!

Felicious reviewed (approved):
Looks great! Ty :D

jero-odoo commented:
Thank you!
@samueljlieber this is ready for tech review.

samueljlieber commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/new_delivery_method.rst:
```diff
@@ -0,0 +1,206 @@
+  For more details on configuring custom shipping methods, such as :ref:`fixed price
+  <inventory/shipping/fixed>`, :ref:`based on rules <inventory/shipping/rules>`, or :ref:`pickup in
+  store <inventory/shipping/pickup>` options, refer to their respective sections below.
+- :guilabel:`Website`: configure shipping methods for an e-commerce page. Select the applicable
```
Please update to bold app name
```suggestion
- :guilabel:`Website`: configure shipping methods for an **eCommerce** page. Select the applicable
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/new_delivery_method.rst:
```diff
@@ -0,0 +1,206 @@
+   .. image:: new_delivery_method/pricing-rule.png
+      :alt: Display window to add a pricing rule. Set a condition and delivery cost.
+
+To restrict shipping to specific destinations on the eCommerce website, in the shipping method form,
+navigate to the :guilabel:`Destination Availability` tab and define the :guilabel:`Countries`,
```
```suggestion
To restrict shipping to specific destinations on the **eCommerce** website, in the shipping method
form, navigate to the :guilabel:`Destination Availability` tab and define the :guilabel:`Countries`,
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/new_delivery_method.rst:
```diff
@@ -0,0 +1,206 @@
+.. figure:: new_delivery_method/shipping-route.png
+   :alt: Routes form with the Shipping Methods checkbox selected.
+
+   Routes form with the Shipping Methods checkbox ticked.
```
Please add a :guilabel: here :)
```suggestion
   Routes form with the :guilabel:`Shipping Method` checkbox ticked.
```

samueljlieber reviewed (approved):
Hi @jero-odoo! Thank you for your work on this PR. Your changes look good to me! Just a couple small fixes please :)
..
@robodoo delegate=jero-odoo

jero-odoo commented:
@robodoo r+

robodoo commented:
@jero-odoo staging failed: ci/runbot on 25ae776e47257223b596c52c8a5f155c8610fb67 (view more at https://runbot.odoo.com/runbot/build/74323111)

robodoo commented:
This staging failed for probably sprurious reasons, we've reset the PR, it should be re-staged soon-ish.
